Content.Server/FeedbackSystem/FeedbackCommand.cs

```diff
@@ -0,0 +1,66 @@
+using Content.Server.Administration;
+using Content.Shared.Administration;
+using Content.Shared.FeedbackSystem;
+using Robust.Shared.Console;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.FeedbackSystem;
+
+[AdminCommand(AdminFlags.Server)]
+public sealed class FeedbackPopupCommand : LocalizedEntityCommands
+{
+    [Dependency] private readonly IPrototypeManager _proto = default!;
+    [Dependency] private readonly SharedFeedbackSystem _feedback = default!;
+
+    public override string Command => Loc.GetString("feedbackpopup-command-name");
+
+    public override void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        if (args.Length != 2)
+        {
+            shell.WriteError(Loc.GetString("feedbackpopup-command-error-wrong-arguments"));
```

Comment:
One of the few times we can get away with reusing loc strings
```suggestion
            shell.WriteError(Loc.GetString("shell-wrong-arguments-number"));
```